tests/integration/checkpoint.bats

```diff
@@ -315,3 +323,45 @@ function simple_cr() {
 	unlink "$tmp"
 	test -f ./work-dir/"$tmplog2" && unlink ./work-dir/"$tmplog2"
 }
+
+@test "checkpoint and restore with nested bind mounts" {
+	CR_TMP=$(mktemp -d)
+	bind1=$(mktemp -d -p "$CR_TMP")
+	bind2=$(mktemp -d -p "$CR_TMP")
```

Comment:
Here every test case is run with cd to a temporary directory containing the bundle (i.e. `rootfs` and `config.json`). This directory can be reused for other things, too, and is removed by `teardown_busybox`, so if you use `mktemp -p .` you don't have to clean up anything.

Reply:
fixed